android-components/components/service/sync-logins/src/main/java/mozilla/components/service/sync/logins/SyncableLoginsStorage.kt

```diff
@@ -248,30 +237,25 @@ class SyncableLoginsStorage(
         conn.close()
     }
 
-    private suspend fun migrateSQLCipherDBIfNeeded() {
-        // Migrate the SQLCipher DB if needed
-        val sqlcipherKey = securePrefs.value.getString(ENCRYPTION_KEY_SQLCIPHER) ?: return
+    /*
+     * We not longer migrate SQLCipher, we delete the DB, key and any
+     * associated prefs
+     */
+    private fun deleteSQLCipherDBIfNeeded() {
```

Comment:
This will also likely need to be suspending as `migrateSQLCipherDBIfNeeded` was, because `getDatabasePath` and `file.delete()` will both be IO operations that need to run off the main thread.

Reply:
good catch! I added back in the suspend and the `runBlocking` at the callsite as well.